translib/test/sflow/openconfig_sflow/05_add_col.json

```diff
@@ -0,0 +1,7 @@
+{"openconfig-sampling-sflow:collector": [
+    {
+     "address": "1.1.1.1",
+     "port": 4444,
+     "network-instance": "default"
```

Comment:
Is this a reference to openconfig's network-instance yang? It is not supported in the community master yet